docs/signaling.md

```diff
@@ -16,3 +16,43 @@ As we're in a P2P network, it is possible to add bad data. However, we wanted to
 
 * [SignalingMessage.recipientSessionId] and [SignalingMessage.senderSessionId] IDs are overwritten before they go to the TomP2P layer, since they could be faked by WebSocket clients
```

Comment:
SignalingMessage.recipientSessionId] and [SignalingMessage.senderSessionId] IDs are overwritten before they go to the TomP2P layer. (I'd remove the rest)